db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_vdc_networks.rb

```diff
@@ -0,0 +1,47 @@
+class AvailableVdcNetworks
+  def initialize(handle = $evm)
+    @handle = handle
+  end
+
+  def main
+    fill_dialog_field(fetch_list_data)
+  end
+
+  def fetch_list_data
+    service = @handle.root.attributes["service_template"] || @handle.root.attributes["service"]
+    vdc_networks = service.try(:orchestration_manager).try(:cloud_networks).try(:select) do |net|
```

Comment:
The `do` block should not fail as it is behind the `try(:select)`. As an example, [this test](https://github.com/ManageIQ/manageiq/pull/11975/files#diff-a16be22b4fbf6fc74854a73654d3eb09R33) doesn't have the `orchestration_manager` set:

```
[4] pry(#<AvailableVdcNetworks>)> vdc_networks = service.try(:orchestration_manager)
=> nil
```

However, the entire `try` block will succeed and return nil:
```
[6] pry(#<AvailableVdcNetworks>)> vdc_networks = service.try(:orchestration_manager).try(:cloud_networks).try(:select) { |net| net.type == "ManageIQ::Providers::Vmware::NetworkManager::CloudNetwork::OrgVdcNet" }
=> nil
```
Consequently the function will [return](https://github.com/ManageIQ/manageiq/pull/11975/files#diff-3a6e2e6a8cb0776d4bd39e2133fce3e7R16) `<none>`.